packages/curriculum-parser-markdown/plugins/insight/yaml.js

```diff
@@ -51,10 +77,16 @@ function getMarkdownLink(link) {
 
 // http://stackoverflow.com/questions/8498592/extract-root-domain-name-from-string
 function getDomainFromURL(url) {
-  return url
-    ? // find & remove protocol (http, ftp, etc.) and get domain
-      (url.indexOf('://') > -1 ? url.split('/')[2] : url.split('/'))
-        // find & remove port number
-        .split(':')
-    : null;
+  if (!url) return null;
+
+  // eslint-disable-next-line node/no-unsupported-features/node-builtins
```

Comment:
I had to disable this rule because it was falling

https://github.com/enkidevs/curriculum-processors/pull/399/commits/153a2a4ef99213006749d158503a825554ce79d3